### DISCUSSION TITLE
files missing from ants_2.3.4_20200811.sif

### DISCUSSION THREAD

**Discussion by Matte16** [neurodesk/neurodesk.github.io#120] [General]
Hi Neurodesk gurus,

I have noticed that there are some files missing from ants_2.3.4_20200811.sif , e.g., antsCookTemplatePriors.sh

In fact, ants_2.3.4_20200811.sif has ~123 files, whereas my local ants installation (albeit 2.3.1) has ~164 files.

Can you look into this, please? 

Regards,

Matt

**Comment by stebo85** [comment]
Dear matt,

Thank you for finding this. I have a look at it today :)

Cheers
Steffen

**Comment by Matte16** [comment]
No worries - thanks Steffan 😉

Matt
________________________________

**Comment by civier** [comment]
I've just checked ants_2.3.4_20210104 and it has around 130, so still not
good.
The version Matt is using is an old one because he's running it on our
supercomputer, and we do not update the packages there often.

Best,
Oren

**Comment by stebo85** [comment]
Dear @Matte16 

ants 2.3.4 doesn't contain antsCookTemplatePriors.sh anymore. It is available in 2.3.1 and I now added this to neurodesk as well. Are you sure that antsCookTemplatePriors.sh is not deprecated and should be available in 2.3.4?

Kind regards
Steffen

**Reply by stebo85** [reply]
@Matte16 - I did a bit more investigation and it looks the script still exists https://github.com/ANTsX/ANTs/tree/master/Scripts but is currently not installed by default. I see if I can change that :)

**Reply by stebo85** [reply]
@Matte16 - I fixed this bug in the latest version of Neurodesk:   vnmd/neurodesktop:20211220

These are the updated singularity containers:
        "apps": {
            "ants 2.3.1": {
                "version": "20211204",
            },
            "ants 2.3.4": {
                "version": "20211212",
            },
            "ants 2.3.5": {
                "version": "20211212",
            }
